datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/GmsGraphQLEngine.java

```diff
@@ -238,6 +242,7 @@ public GmsGraphQLEngine(
         this.mlPrimaryKeyType = new MLPrimaryKeyType(entityClient);
         this.dataFlowType = new DataFlowType(entityClient);
         this.dataJobType = new DataJobType(entityClient);
+        this.entityRegistry = entityRegistry;
```

Comment:
NIT: seems odd to place this here. Can we put it below all of the types? Or above?